wordpress-mobile/WordPress-iOS
# Update WordPressAuthenticator-iOS pod

Fixes #11406. Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/71

## To test:
0. Have a self-hosted site credentials ready.
1. Log out of the app
2. Check out this branch
3. Run the build and follow the Login via Site URL flow (use the "Log in by entering your site address" button)
4. Enter the username / password for the site. 
5. Expected behavior: the Login via Site URL flow has not changed. No new errors appear. Everything works as before.

Update release notes:
